doc/rdesktop.1

```diff
@@ -123,6 +128,11 @@ but everything after this is unencrypted (including interactive logins).
 Do not send mouse motion events.  This saves bandwidth, although some Windows
 applications may rely on receiving mouse motion.
 .TP
+.BR "-M"
```

Comment:
You should break out local / remote mouse rendering from this pull request into one of its own.